### PR TITLE
Fix slash command focus style.

### LINKED ISSUE #23718: Inconsistencies with G2 select/focus states
I was testing the latest G2 styling and came up with a few small inconsistencies with focus/select states (border widths and radius) on items (screenshots below). I'm thinking we should have consistent border widths, colors and radii. Thoughts?

Screen Shot 2020-07-06 at 1 19 13 PM

Screen Shot 2020-07-06 at 1 19 07 PM

Screen Shot 2020-07-06 at 1 19 35 PM

Screen Shot 2020-07-06 at 1 33 25 PM
 
Screen Shot 2020-07-06 at 1 33 19 PM

### PR DESCRIPTION
**Pull request by jasmussen**
## Description

Fixes #23718.

The focus style width variable was not used in the auto complete component, meaning it was wrong:

Screenshot 2021-06-30 at 08 33 02

This PR makes it right:

Screenshot 2021-06-30 at 08 34 21

I also verified that the border radius is correct compared to other elements.

## How has this been tested?

Use the slash command and verify the focus style is right.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] I've tested my changes with keyboard and screen readers. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal).

### CODE REVIEW

**Review by Mamaduka — APPROVED** [review]
LGTM 👍

**Comment by jasmussen** [comment]
Thank you! There are some failing tests that are unrelated, so I will wait for them to be fixed in trunk, then rebase, before I merge.

**Comment by Mamaduka** [comment]
I restarted the tests. I think the pattern issue should be fixed for now. Let's hope others don't fail 🤞

**Pull request merged**
